### PR TITLE
feat: [PLG-2419]: Added telemetry and K8s swimlanes

### PR DESCRIPTION
**Pull request by devinder-harness**
### Summary

Added:
* Telemtry
* K8s Swimlanes
  * Kustomize
  * Helm with forks
 * Serverless Swimlanes (Disabled in this PR)
 * Links to doc
 * Changed installation instructions for `Mac` and `Linux` 
 

JIRA : https://harness.atlassian.net/browse/PLG-2419
#### Screenshots
image
image
image
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by sahithikolichala — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts [hunk 1]
```diff
@@ -25,8 +38,65 @@ export const INFRA_TYPES: { [key: string]: EntityMap } = {
...
+    GCPFunction: {
+      id: SERVERLESS_FUNCTIONS.GOOGLE_CLOUD_FUNCTION,
+      label: 'Google Cloud Function',
+      icon: 'service-google-functions'
+    }
+  },
+  TraditionalApp: {
+    TraditionalAWS: {
+      id: 'TraditionalAWS',
+      label: 'Traditional AWS',
```
Comment (reetika04) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts:58]: Can we use strings for labels?

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts [hunk 2]
```diff
@@ -4,19 +4,32 @@
...
   KubernetesService: { id: 'KubernetesService', label: 'Kubernetes Service', icon: 'app-kubernetes' },
-  ServerslessFunction: { id: 'ServerlessFunction', label: 'Serverless Function', icon: 'serverless-deploy-step' },
-  TraditionalApp: { id: 'TraditionalApp', label: 'Traditional App', icon: 'square' }
+  ServerlessFunction: {
+    id: 'ServerlessFunction',
+    label: 'Serverless Function',
+    icon: 'serverless-deploy-step',
+    className: css.blueIcons
+  },
+  TraditionalApp: { id: 'TraditionalApp', label: 'Traditional App', icon: 'chaos-cube', className: css.blueIcons }
```
Comment (reetika04) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts:26]: strings for labels?

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/AWS/ConfigureAWS.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+          {getString('cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.configureAWSStep.uploadSVCKey')}
+        </Label>
+        <TextInput
+          id="gcpservicekeyfile"
+          name="gcpservicekeyfile"
+          defaultValue={''}
+          placeholder={getString(
+            'cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.configureGCPStep.placholders.bucketName'
+          )}
+          // onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (reetika04) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/AWS/ConfigureAWS.tsx:64]: Is this needed?

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts (hunk 2 above)
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts:26]: this is WIP , will move them in followup PR

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/AWS/ConfigureAWS.tsx (hunk 3 above)
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/AWS/ConfigureAWS.tsx:64]: same as above WIP feature

**Review by reetika04 — APPROVED** [review]

**Comment by devinder-harness** [comment]
Coverage to be picked up in future sprints as https://pbs.twimg.com/media/FBqoOhpVIBcRzyr.jpg

**Pull request merged**